internal/listener/listener.go

```diff
@@ -133,7 +134,14 @@ func (l *Listener) ProcessEvent(w http.ResponseWriter, req *http.Request) {
 
 	l.logger.Infow("Processing event", zap.String("event", ev.String()))
 	err = incident.ProcessEvent(context.Background(), l.db, l.logs, l.runtimeConfig, &ev)
-	if err != nil {
+	switch {
+	case errors.Is(err, event.ErrSuperfluousStateChange):
+		fallthrough
+	case errors.Is(err, event.ErrSuperfluousMuteUnmuteEvent):
+		abort(http.StatusNotAcceptable, &ev, "%v", err)
+		return
+
+	case err != nil:
```

Comment:
I already noticed that you're a way bigger fan of `switch` than I am, but with the `fallthrough` here, a regular `if` looks way nicer to me:
```suggestion
	if errors.Is(err, event.ErrSuperfluousStateChange) || errors.Is(err, event.ErrSuperfluousMuteUnmuteEvent) {
		abort(http.StatusNotAcceptable, &ev, "%v", err)
		return
	} else if err != nil {
```

Reply:
Fair enough.